valhalla/thor/costmatrix.h

```diff
@@ -164,11 +164,11 @@ class CostMatrix : public MatrixAlgorithm {
   baldr::DateTime::tz_sys_info_cache_t tz_cache_;
 
   /**
-   * Get the cost threshold based on the current mode and the max arc-length distance
-   * for that mode.
+   * Get the cost range for the doublebucket queue based on the current mode and the max arc-length
+   * distance for that mode.
    * @param  max_matrix_distance   Maximum arc-length distance for current mode.
    */
-  float GetCostThreshold(const float max_matrix_distance);
+  float GetBucketRange(const float max_matrix_distance);
```

Comment:
you can just remove this. seems like you intended to replace it at first and then inline you just did the simple math instead, no function call needed